
ï»¿src.ai.models.twelve_labs.exceptions.TwelveLabsAuthError

==========================================================










.. currentmodule:: src.ai.models.twelve_labs.exceptions

.. autoexception:: TwelveLabsAuthError
